platform/src/main/java/org/hillview/storage/GenericLogs.java

```diff
@@ -42,6 +44,15 @@
     private Schema schema;
     private Grok grok;
     private String logFormat;
+    private String multiLine;
```

Comment:
isn't it more natural to keep these in the inner class?

Reply:
Sure, will move to inner class